commitment/commitment_test.go

```diff
@@ -329,8 +342,28 @@ func TestMintTapCommitment(t *testing.T) {
 	for _, testCase := range testCases {
 		success := t.Run(testCase.name, func(t *testing.T) {
 			details := testCase.f()
-			groupKey := asset.RandGroupKey(t, testCase.g)
-			_, _, err := Mint(testCase.g, groupKey, details)
+			// Test cases with a nil amount should fail in the call
+			// to Mint() below, not on asset creation. Use the safe
+			// amount of 1 here, and pass the original amount to
+			// Min() below.
+			var amt uint64
```

Comment:
Why wasn't this the case before?

Reply:
Should be resolved - we just want invalid amounts to fail because of a call to `Mint()` and not another function used for test setup.